### PR TITLE
Add YAML ABI draft

### PR DESCRIPTION
**Pull request by axic**
It seems I've never pushed this informational EIP :confused:

### CODE REVIEW

File: EIPS/eip-draft_yaml_abi.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+In the past, most suggestions included an ABI description purely as a Solidity contract and/or interface. This has several drawbacks:
+- Prefers a single language over others and could hinder the development of new languages.
+- Locks the specification to a certain version of the Solidity language.
+- Allows syntactical elements and features of the language, which may not be well representable in the ABI.
+
+This proposal aims to solve all these issues.
+
+## Specification
+
+The *Standard Contract ABI* is usually represented as a JSON object.  This works well and several tools – including compilers and clients – support it to handle data encoding.
```
Comment (axic) [inline EIPS/eip-draft_yaml_abi.md]: Should actually merge #1605 and refer to it here under "Standard Contract ABI".

**Comment by tjayrush** [comment]
Can you add a discussion link to this EIP at the magician's site? I'd like to make a few comments on it, but there's no official discussion link. If you want, I'll make a PR. Not sure what the right procedure is.

**Comment by axic** [comment]
@tjayrush this is still in unfinished draft mode (need to polish it a bit), but created a URL here: https://ethereum-magicians.org/t/eip-2069-recommendation-for-using-yaml-abi-in-specifications/3347

**Comment by axic** [comment]
@Arachnid @nicksavers can this be merged as a draft?

**Review by jasonborn05 — COMMENTED** [review]
Nanobotsbitc

**Review by jasonborn05 — COMMENTED** [review]
Nanobotsbitc

**Pull request merged**